packages/cli/src/commands/setup/deploy/providers/serverless.js

```diff
@@ -0,0 +1,121 @@
+// import terminalLink from 'terminal-link'
+import fs from 'fs'
+import path from 'path'
+
+import Listr from 'listr'
+
+import { errorTelemetry } from '@redwoodjs/telemetry'
+
+import { getPaths } from '../../../../lib'
+import c from '../../../../lib/colors'
+import {
+  addToGitIgnoreTask,
+  addFilesTask,
+  addPackagesTask,
+  printSetupNotes,
+} from '../helpers'
+import { SERVERLESS_API_YML } from '../templates/serverless/api'
+import { SERVERLESS_WEB_YML } from '../templates/serverless/web'
+
+export const command = 'serverless'
+export const description = 'Setup deployments via the Serverless Framework'
+
+export const aliases = ['aws-serverless']
+
+export const notes = [
+  `${c.green(
+    "You're almost ready to deploy using the Serverless framework!"
+  )}\n`,
+  '• See https://redwoodjs.com/docs/deploy#aws_serverless for more info, including ',
```

Comment:
@cannikin should we mention running deploy with `--first-run` flag here? Also should we call the flag --first-deploy?

Reply:
Good call, I'll also update that doc link since it won't be `aws_serverless` any more.

Reply:
I borrowed `--first-run` from Rails, of course. ;) You're already running the `deploy` command so seems kind of redundant to have it in the flag name again? But I could do either way, whichever you think is clearer.

Reply:
Okay updated that text.